systemjs.config.js

```diff
@@ -34,9 +34,10 @@
     'node_modules/systemjs-plugin-babel/systemjs-babel-browser.js';
   var PluginBabelCachePath = 'external/systemjs/plugin-babel-cached.js';
 
-  var isCachingPossible = typeof indexedDB !== 'undefined' &&
-                          typeof TextEncoder !== 'undefined' &&
-                          typeof crypto !== 'undefined';
+  var isCachingPossible = (typeof indexedDB !== 'undefined') &&
+                          (typeof TextEncoder !== 'undefined') &&
+                          (typeof crypto !== 'undefined') &&
+                          (typeof crypto.subtle !== 'undefined');
```

Comment:
Nit: There's really no need to add parenthesis on every line here, it's quite readable anyway.
(Also, in this case, it unnecessarily changes the blame for unrelated lines.)